# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/kmod/hotfix-missing-header.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Fixes segfault caused by basename() returning an int instead of a char pointer.

  - NoTag <notag@t2sde.org>

--- kmod-32/tools/kmod.c.vanilla	2024-03-20 12:36:04.308777149 +0100
+++ kmod-32/tools/kmod.c	2024-03-20 14:32:24.947211013 +0100
@@ -22,6 +22,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <libgen.h>
 
 #include <shared/util.h>
 
